Makefile

```diff
@@ -65,7 +65,7 @@ test:
 	go vet ./...
 
 prepare:
-	glide install
+	glide update && glide install
```

Comment:
Hmm I am seeing the same failure as before.  Can you remove the `glide install` here and just have `glide update` as a check?